src/core/io_queue.cc

```diff
@@ -551,7 +555,6 @@ void
 io_queue::complete_request(io_desc_read_write& desc) noexcept {
     _requests_executing--;
     _requests_completed++;
-    _streams[desc.stream()].notify_request_finished(desc.capacity());
```

Comment:
It seems, that the function does not need to take a parameter -- it is just responsible for accounting.

Reply:
True